reps/2022-04-07-serialization.md

```diff
@@ -0,0 +1,162 @@
+## Summary
+### General Motivation
+Current Ray's serialization has some issues:
+
+1. Doesn't support out-of-band serialization in Java worker. Cause some perfomance issues in Java worker.
+1. Hard to add a new serializer for a specific class.
+1. Doens't support cross-language serialization for custom classes.
```

Comment:
Besides custom classes. There are other issues as well. For example: 1) type loss (Java's int/short/byte); 2) support for maps. Also, let's put more examples here.